.gitlab/deploy_7/docker.yml

```diff
@@ -53,6 +53,7 @@ tag_release_7_windows_docker_hub:
   variables:
     VARIANT: 2004
   script:
+    - '$_instance_id = (iwr  -UseBasicParsing http://169.254.169.254/latest/meta-data/instance-id).content ; Write-Host "Running on instance $($_instance_id)"'
```

Comment:
Is this needed, given that the instance id is already printed in `.docker_tag_windows_job_definition` (which this job is an extension of)?